courses/serializers.py

```diff
@@ -334,6 +335,23 @@ def get_topics(self, instance):
         )
         return list(topics)
 
+    def get_requirements(self, instance):
```

Comment:
Hmm. Not sure - I named it requirements because the model is `ProgramRequirement`. You have to satisfy the electives to be able to complete the program too, though, I thought.

Reply:
Your explanation makes sense to me.